servers/pvprefuge/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "id": "pvprefuge",
+  "name": "PvPRefuge",
+  "description": "PvPRefuge is a network focused on PvP like UHC, HCF & Practice.",
+  "store": "https://store.pvprefuge.net/",
+  "addresses": [
+    "pvprefuge.net"
+  ],
+  "primaryAddress": "pvprefuge.net",
+  "minecraftVersions": [
+    "1.7",
+    "1.8"
```

Comment:
We've updated the way we handle minecraftVersions. Please check the README.MD for more info.